presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/HyperLogLogStateSerializer.java

```diff
@@ -19,15 +19,15 @@
 import com.facebook.presto.spi.type.Type;
 import io.airlift.stats.cardinality.HyperLogLog;
 
-import static com.facebook.presto.spi.type.HyperLogLogType.HYPER_LOG_LOG;
+import static com.facebook.presto.spi.type.VarbinaryType.VARBINARY;
 
 public class HyperLogLogStateSerializer
         implements AccumulatorStateSerializer<HyperLogLogState>
 {
     @Override
     public Type getSerializedType()
     {
-        return HYPER_LOG_LOG;
```

Comment:
Could you help me understand why we change this?

Reply:
The hive connector (that we use for materialization) doesn't know anything about the `HYPER_LOG_LOG` type, so it cannot serialize the intermediate aggregation results. But in fact it is not needed to have the intermediate type of the `HyperLogLogStateSerializer`, as it anyhow serializes the intermediate state to binary representation.